wire-library/wire-kotlin-generator/src/test/java/com/squareup/wire/kotlin/KotlinGeneratorTest.kt

```diff
@@ -147,25 +147,29 @@ class KotlinGeneratorTest {
   }
 
   @Test fun blockingSingleRequestSingleResponse() {
+    //language=kotlin
```

Comment:
What are those comments? I don't remember seeing them.

Reply:
should we get rid of them?

Reply:
This hints the IDE to add Kotlin highlighting to the multiline string, which makes it nicer to read/edit those fixtures in the IDE.